src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AuditingTests.ps1

```diff
@@ -1098,3 +1098,38 @@ function Test-ServerAuditingTypeMigration
 		Remove-TestEnvironment $testSuffix
 	}
 }
+
+<#
+.SYNOPSIS
+Tests that Get Server and Database Auditing on UK passes
+#>
+function Test-GetServerAndDatabaseAuditingInUkRegion
+{
+	# Setup
+	$testSuffix = 684412
+	Create-TestEnvironment $testSuffix "UK South"
+	$params = Get-SqlAuditingTestEnvironmentParameters $testSuffix 
+
+	try
+	{                              
+		# Test
+		$policy = Get-AzureRmSqlServerAuditingPolicy -ResourceGroupName $params.rgname -ServerName $params.serverName
+                                
+        # Assert
```

Comment:
@yaakoviyun can you fix the indentation in the changes you made to this file?

Other than that, LGTM